specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2016-10-01/examples/checkVaultNameAvailability.json

```diff
@@ -0,0 +1,17 @@
+{
+    "parameters":{
+       "subscriptionId":"00000000-0000-0000-0000-000000000000",
+       "api-version":"2016-10-01",
+       "parameters":{
+           "name":"sample-vault",
+           "type":"Microsoft.KeyVault/vaults"
+       }
```

Comment:
Need body parameter of type "VaultCheckNameAvailabilityParameters".

Reply:
This does match the scheme of VaultCheckNameAvailabilityParameters

Reply:
You need to put it in "vaultName": {}